src/backend/x64/a64_emit_x64.cpp

```diff
@@ -877,51 +934,76 @@ void A64EmitX64::EmitDirectPageTableMemoryWrite(A64EmitContext& ctx, IR::Inst* i
 }
 
 void A64EmitX64::EmitA64ReadMemory8(A64EmitContext& ctx, IR::Inst* inst) {
-    if (conf.page_table) {
+    if (conf.fastmem_pointer) {
+        EmitDirectMemoryRead(ctx, inst, 8);
+    }
+    else if (conf.page_table) {
```

Comment:
Our formatting combines the braces onto one line like so:

```suggestion
    } else if (conf.page_table) {
```

ditto for all other occurrences